django/librarian/templates/librarian/components/document_list.html

```diff
@@ -31,26 +32,83 @@ <h6 id="documents-label">{% trans "Documents" %}</h6>
       </div>
     </div>
     <div class="row mb-2">
-      <div class="col pe-0">
-        <button type="button"
-                class="btn btn-sm btn-outline-secondary w-100"
-                hx-get="{% url 'librarian:data_source_stop' selected_data_source.id %}">{% trans "Stop all" %}</button>
-      </div>
-      <div class="col ps-1 pe-0">
-        <button type="button"
-                class="btn btn-sm btn-outline-success w-100"
-                hx-get="{% url 'librarian:data_source_start' selected_data_source.id %}">
-          {% trans "Process all" %}
-        </button>
-      </div>
-      <div class="col ps-1">
-        <button type="button"
-                class="btn btn-sm btn-outline-success w-100"
-                hx-get="{% url 'librarian:data_source_start_azure' selected_data_source.id %}">
-          {% trans "Force OCR" %}
-        </button>
+      <div class="col">
+        <div class="btn-group w-100">
+          <button type="button"
+                  class="btn btn-sm btn-outline-success dropdown-toggle"
+                  data-bs-toggle="dropdown"
+                  aria-expanded="false">{% trans "Bulk actions" %}</button>
+          <ul class="dropdown-menu">
+            <li>
+              <h6 class="dropdown-header">{% trans "Process all, including completed" %}</h6>
+            </li>
+            <li>
+              <a class="dropdown-item small"
+                 href="#"
+                 hx-get="{% url 'librarian:data_source_start' selected_data_source.id 'default' 'all' %}">
+                <span>{% trans "Text only" %}</span>
+                <span class="text-muted">{% trans "(default, $)" %}</span>
+              </a>
+            </li>
+            <li>
+              <a class="dropdown-item small"
+                 href="#"
+                 hx-get="{% url 'librarian:data_source_start' selected_data_source.id 'azure_read' 'all' %}">
+                <span>{% trans "PDF OCR" %}</span>
+                <span class="text-muted">{% trans "(more robust, $$)" %}</span>
+              </a>
+            </li>
+            <li>
+              <a class="dropdown-item small"
+                 href="#"
+                 hx-get="{% url 'librarian:data_source_start' selected_data_source.id 'azure_layout' 'all' %}">
+                <span>{% trans "PDF layout & OCR" %}</span>
+                <span class="text-muted">{% trans "(best, $$$)" %}</span>
+              </a>
+            </li>
+            <li>
+              <hr class="dropdown-divider">
+            </li>
+            <li>
+              <h6 class="dropdown-header">{% trans "Process incomplete only" %}</h6>
+            </li>
```

Comment:
Not very clear what "process incomplete only" means, I'm guessing it means it will process the files that have failed before?